### PR TITLE
Document example systemd service configuration

### PR DESCRIPTION
**Pull request by larry-cdn77**
In response to https://github.com/ContentSquare/chproxy/pull/378#pullrequestreview-1805872171

### CODE REVIEW

File: docs/src/content/docs/install.md [hunk 1]
```diff
@@ -67,3 +67,31 @@ Example:
...
+```
+[Unit]
+Description=chproxy
+Requires=network.target
+After=network.target
+
+[Service]
+Type=notify
+User=root
+Group=root
```
Comment (vfoucault) [inline docs/src/content/docs/install.md]: Not a good idea to recommend running as root by default

```suggestion
User=chproxy
Group=chproxy
```

File: docs/src/content/docs/install.md [hunk 2]
```diff
@@ -67,3 +67,31 @@ Example:
...
 docker run -d -v $(PWD)/config.yml:/opt/config.yml -p 9090:9090 chproxy-test -config /opt/config.yml
 ```
+
+### systemd service
+
+Example systemd service configuration:
+
+```
+[Unit]
+Description=chproxy
```
Comment (vfoucault) [inline docs/src/content/docs/install.md]:
```suggestion
Description=CHProxy
Documentation=https://www.chproxy.org/
```

**Review by vfoucault — COMMENTED** [review]
As a best practice, avoid running as root when not needed.

File: docs/src/content/docs/install.md (hunk 1 above)
Comment (larry-cdn77) [reply docs/src/content/docs/install.md]: Thanks, changed 👍

File: docs/src/content/docs/install.md (hunk 2 above)
Comment (larry-cdn77) [reply docs/src/content/docs/install.md]: Thank you, added the documentation link

I looked for refererences to capitalisation of chproxy and found different versions, the GitHub landing page (readme) reads 'chproxy', at chproxy.org I see 'Chproxy' while some source/config comments uses 'CHProxy'?

Comment (vfoucault) [reply docs/src/content/docs/install.md]: I've checked, and the correct spelling is `Chproxy`. Thanks for your time and contributions 🤗

Comment (larry-cdn77) [reply docs/src/content/docs/install.md]: Okay, sounds good to me 🙂 Changed to Chproxy

Thank you for the review

**Review by mga-chka — APPROVED** [review]

**Pull request merged**